### PR TITLE
Mobile finish portfolio project.

### PR DESCRIPTION
**Pull request by TumiRantsane**
Changes and advancement done to meet alignment issues for a mobile device.

### CODE REVIEW

**Review by Lizdev-05 — APPROVED** [review]
Hi @TumiRantsane, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved

## Highlights
- Descriptive commit messages
- No linters error
- Project requirements are met
## Optional suggestions
- Please, don't open a new pull request when you are requested to make changes to avoid confusion.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    </section>
+    <section class="contact" id="contact">
+      <div class="contact__container">
+        <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
+        <p class="contact__dsc fw-500 fz-20 lh-28">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I'd love
+          to help with it
+        </p>
+        <form id="form" action="#" method="post">
```
Comment (ha-manel) [inline index.html:336]: - [ ] Kindly add the link to `formspree` in the `action` attribute. it should be similar to this `https://formspree.io/f/{form_id}`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+          <ul class="ul">
+            <li>
+              <label for="fname" class="fw-500 fz-17 lh-20">Name:</label>
```
Comment (ha-manel) [inline index.html:339]: - [ ] The form is looking good, however we should follow the design in Figma as much as possible. Kindly remove the labels to match the design 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @TumiRantsane👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    </section>
+    <section class="contact" id="contact">
+      <div class="contact__container">
+        <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
+        <p class="contact__dsc fw-500 fz-20 lh-28">
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I'd love
+          to help with it
+        </p>
+        <form id="form" action="#" method="post">
```
Comment (anagudelogu) [inline index.html:336]: - [ ] As requested by the previous reviewer, please use the [Formspree](https://formspree.io/html/) service to handle your contact's form submissions. 📜 It is important to be able to collect data from people that are trying to connect with us! Please, implement this. 🤓 🛠️ 

Requirements:
image

**Review by anagudelogu — CHANGES_REQUESTED** [review]
Hi @TumiRantsane, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.



 ## Highlights
✅ Descriptive commit messages. 👍🏼
✅ Great use of GitHub flow. 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
# Hi @TumiRantsane 🟢 🟢 🟢

## In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it 🥳 🥳 🥳


## Good luck! 🎉 🎉



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**